packages/conf-allegro5/conf-allegro5.1/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "allegro-developers@lists.liballeg.org"
+homepage: "https://liballeg.org"
+authors: "The Allegro authors"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "Giftware"
+build: ["pkg-config" "allegro-5"]
+depexts: [
+  [
+    "allegro5"
+    "allegro5-addon-image"
+    "allegro5-addon-primitives"
+    "allegro5-addon-ttf"
+  ] {os-distribution = "centos" | os-distribution = "fedora"}
+  [
+    "liballegro-devel"
+    "liballegro_image-devel"
+    "liballegro_primitives-devel"
+    "liballegro_ttf-devel"
+  ] {os-family = "suse" | os-family = "opensuse"}
+  [
+    "liballegro5-dev"
+    "liballegro-image5-dev"
+    "liballegro-primitives5-dev"
+    "liballegro-ttf5-dev"
+  ] {os-family = "debian" | os-family = "ubuntu"}
+  ["allegro5"] {os-distribution = "nixos"}
+  ["allegro"] {os-family = "arch"}
+  ["allegro-dev"] {os-family = "alpine"}
+  ["allegro"] {os-distribution = "homebrew"}
+  ["allegro5"] {os = "freebsd"}
+  ["allegro5"] {os = "netbsd"}
```

Comment:
It looks like we might get some windows packaging support via msys2, via https://packages.msys2.org/base/mingw-w64-allegro

If you're interested in giving that a shot, you could add

```suggestion
  ["allegro5"] {os = "netbsd"}
  ["mingw-w64-allegro"] {os-distribution = "msys2"}
```

Of course, you don't need to do everything at once, so if you'd just like us to move forward with publising this as is, I think that is fine!